moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -1550,6 +1595,86 @@ bool MoveItConfigData::setPackagePath(const std::string& pkg_path)
   return true;
 }
 
+// ******************************************************************************************
+// Extract the package/stack name from an absolute file path
```

Comment:
This whole function is rather complex. But I just have seen that you moved it from `StartScreenWidget`.
May I ask for some changes anyway?

Reply:
Sure, I just didn't want to put too much into one PR